### PR TITLE
[FieldCollection] Json import ignores title and group

### PR DESCRIPTION
**Pull request by blankse**

### CODE REVIEW

**Comment by brusch** [comment]
Thanks a lot!

File: models/DataObject/ClassDefinition/Service.php [hunk 1]
```diff
@@ -149,14 +149,19 @@ public static function importFieldCollectionFromJson($fieldCollection, $json, $t
             $fieldCollection->setLayoutDefinitions($layout);
         }
 
-        $fieldCollection->setParentClass($importData['parentClass']);
+        foreach (['parentClass', 'title', 'group'] as $importPropertyName) {
+            if (isset($importData[$importPropertyName])) {
+                $fieldCollection->{'set' . $importPropertyName}($importData[$importPropertyName]);
```
Comment (yariksheptykin) [inline models/DataObject/ClassDefinition/Service.php]: @blankse Isn't `ucfirst` missing here?
i. e:
```
$fieldCollection->{'set' . ucfirst($importPropertyName)}($importData[$importPropertyName]);
```
It used to be `$fieldCollection->setParentClass`. Now it's become `$fieldCollection->setparentClass`. I'd expect a "missing method" error.

Comment (blankse) [reply models/DataObject/ClassDefinition/Service.php]: @yariksheptykin Yes ucfirst is missing. Does it work in class import? There is the same:
https://github.com/pimcore/pimcore/blob/master/models/DataObject/ClassDefinition/Service.php#L112

Comment (yariksheptykin) [reply models/DataObject/ClassDefinition/Service.php]: This is a good question. To my understanding that should not work either. Is there any magic that could convert `setparentClass` to `setParentClass` in AbstractModel / AbstractDao, @brusch ?

Comment (blankse) [reply models/DataObject/ClassDefinition/Service.php]: @yariksheptykin I checked it. Function names are case-insensitive in PHP.
See:
- https://www.php.net/manual/en/functions.user-defined.php
- https://stackoverflow.com/questions/33273941/php-case-sensitivity

I think we should nevertheless use the correct spelling

Comment (brusch) [reply models/DataObject/ClassDefinition/Service.php]: @blankse @yariksheptykin added `ucfirst()` thanks for reviewing!

**Pull request merged**